lms/envs/common.py

```diff
@@ -1860,6 +1860,9 @@
     # Microsite configuration
     'microsite_configuration',
 
+    # RSS Proxy
+    'rss_proxy',
```

Comment:
I think the general philosophy of "adding to the [perceived] sprawl" is what we want to think more about. The djangoapp code organization unit offers the component packaging that you are looking for in many ways. If a djangoapp (or other arbitrary code) needs to be reusable across multiple (more than 2) IDA's, then it makes sense to split that code out into its' own versioned package. I think this rss_proxy djangoapp does not fall into that category at this point. That said, I do think we will need to move it to the reorganized version of the marketing/onboarding app once we get that rolling.